modules/installing-using-agent-based-installer-oci.adoc

```diff
@@ -0,0 +1,251 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-agent-based-installer.adoc [Using the Agent-based Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: CONCEPT
+[id="installing-using-agent-based-installer-oci_{context}"]
+= Using the Agent-based Installer to install a cluster on OCI
+
+You can harness all the advantages of the Agent-based installer to install an {product-title} cluster on {oci-first}. The Agent-based installation comprises a bootable ISO that contains the Assisted discovery agent and the Assisted Service. Both of this components are required to perform the cluster installation, but the latter component runs on only one of the hosts.
+
+:FeatureName: Using the Agent-based Installer to install an {product-title} cluster on OCI
+include::snippets/technology-preview.adoc[]
+
+The Agent-based installer provides the ease of use of the Assisted Installation service, but with the capability to install a cluster in either a connected or disconnected environment. Additionally, you can use the Agent-based Installer to generate or accept Zero Touch Provisioning (ZTP) custom resources.
+
+[NOTE]
+====
+Consider selecting a nonvolatile memory express (NVMe) drive or a solid-state drive (SSD) for your boot disk, because these drives offer low latency and high throughput capabilities for your boot disk. 
+====
+
+Before you install {product-title} on {oci}, you must complete some required steps for creating an {oci} environment on your virtual machine (VM) shape or bare-metal shape. By completing these steps, you can install {product-title} and deploy a cluster on infrastructure that supports a wide range of cloud options and strong security policies.
+
+.Prerequisites
+* You have prior knowledge of {oci} components. See link:https://docs.oracle.com/en-us/iaas/Content/GSG/Concepts/concepts.htm[Learn About Oracle Cloud Basics] in the Oracle documentation.  
+* You reviewed details about the xref:../../architecture/architecture-installation.html#installation-overview_architecture-installation[{product-title} installation and update processes].
+//* You completed the link:https://docs.google.com/forms/d/e/1FAIpQLSdQHmw0BUWjQxPKC5_G7lk2_o9Pcnn2ON84Al6s2Ir254gv6Q/viewform[Request Access to Red Hat OpenShift on {oci} in Developer Preview] form. 
```

Comment:
SME QUERY: Form still needed for TP feature?

Reply:
Please confirm with Ramon.